ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -286,9 +301,9 @@ prop_Mempool_Capacity mcts = withTestMempool mctsTestSetup $
             -- Indicate that we've removed the transactions from the mempool.
```

Comment:
Actually about L279 (not part of this PR), I would use `unless`.

Reply:
Yeah, that would be nicer than `if ... then pure () else ...`